src/api/api.js

```diff
@@ -363,9 +363,18 @@ export function getAllTags(search, pagination = {}) {
   );
 }
 
-export function getOperatingSystems(params = []) {
-  return systemProfile.apiSystemProfileGetOperatingSystem(...params);
-}
+export const getOperatingSystems = async (params = [], showCentosVersions) => {
+  let operatingSystems = await systemProfile.apiSystemProfileGetOperatingSystem(
+    ...params
+  );
+  if (!showCentosVersions) {
+    let newResults = operatingSystems?.results.filter(
```

Comment:
This would error if `operatingSystems?.results` evaluates to undefined. `results` must be always present (this is the part of all of the API standardised responses).